tests/integration/linux_bridge_test.py

```diff
@@ -76,6 +77,16 @@ def bridge0_with_port0(port0_up):
         yield deepcopy(desired_state)
 
 
+@pytest.fixture
+def port0_vlan101(port0_up):
+    vlan_id = 101
+    vlan_base_iface = port0_up[Interface.KEY][0][Interface.NAME]
+    port_name = vlan_base_iface + '.' + str(vlan_id)
```

Comment:
Please use `port_name = '{}.{}'.format(vlan_base_iface, vlan_id)` instead of string concatenation.